docs/application/dotnet/guides/location-sensors/device-sensors.md

```diff
@@ -291,9 +297,62 @@ The following table lists the measurement data that the HRM sensor provides.
 
 | Measurement | Type                 | Range                          | Unit         |
 |-----------|--------------------|------------------------------|------------|
+| Timestamp  | `ulong`    | -                              | Microseconds |
 | TimeSpan   | `TimeSpan` | -                              | Microseconds |
 | HeartRate   | `int`                | Min. value = 0<br>Max. value = 240 | -            |
 
+<a name="hrm_batch"></a>
+## Heart Rate Monitor Batch Sensor
+
+The Heart Rate Monitor (HRM) batch sensor measures a person's heart rate. The sensor must be power-efficient and support batch processing.
```

Comment:
It is recommended to avoid apostrophes in the technical documents. However, throughout the file, I see that in many places, sentences with apostrophes are used. So removing apostrophe from here can bring inconsistency on this page.
Therefore, here we can leave it as it is to avoid consistency issues.

@bwikbs Please do not change anything.

Same comment at Line No. 332

Reply:
@mijin85cho your thoughts on this. Please share your opinion regarding the same.